docs/tables/crowdstrike_detect.md

```diff
@@ -0,0 +1,29 @@
+# Table: crowdstrike_detect
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
```

Comment:
Can you please add examples so each doc has 4-5 examples? For reference, this AWS table has some good examples, https://hub.steampipe.io/plugins/turbot/aws/tables/aws_backup_framework, and there are some more sample guidelines in https://steampipe.io/docs/develop/writing-example-queries#sample-guidelines